SWIPs/swip-honey_to_money.md

```diff
@@ -9,29 +9,29 @@ Created: 31-07-2019
 ## Simple summary 
-Swarm needs a mechanism to allow fluctuation of the exchange rate between honey (Swarms internal accounting unit) and the currency used to pay. We propose an on-chain price oracle that returns an exchange rate + validUntil tuple.  
+Swarm needs a mechanism to allow fluctuation of the exchange rate between honey (Swarms internal accounting unit) and the currency used to pay. We propose an on-chain price oracle that returns an `(exchange rate,validUntil)` tuple.  
 
 ## Abstract 
 The ability to update prices is required in order to:
 * Facilitate experimentation with the absolute price of honey (e.g. start at a low price and gradually increase);
-* Off-set fluctuations of the currency used to pay;
-* Lower the price of Swarm in the long-run as the costs of operating a Swarm node decreases.
+* Offset fluctuations of the currency used to pay;
+* Lower the price of Swarm in the long run as the costs of operating a Swarm node decreases.
 
 We propose an on-chain price oracle that returns the current exchange rate when queried. The price-oracle will be initially managed by Swarm developers and stakeholders.
 
 This SWIP is part of a series of SWIPs (but can be implemented on it's own). To see the full picture, please refer to [swip-message_to_honey](./swip-message_to_honey.md), [swip-multiple](./swip-honey_to_money.md), [swip-multiple_payment_processing_support](./swip-multiple_payment_processing_support.md) and the diagram below:
 ![SWIP_Diagrams.svg](./../assets/swip-honey_to_money/SWIP_Diagrams.svg)
 
 ## Motivation
-Nodes keep track of the balances with each other nodes using an imaginary currency: honey. Since honey is not a currency in which nodes can settle their balances with each other - it is merely a unit of account - there is a requirement to facilitate converting honey to a currency. Furthermore, it is desirable that this conversion rate can fluctuate since:
+Nodes keep track of the balances with each other nodes using an internal reference unit: honey. Since honey is not a currency in which nodes can settle their balances with each other - it is merely a unit of account - there is a requirement to facilitate converting honey to a currency. Furthermore, it is desirable that this conversion rate can fluctuate since:
 
 * The Swarm developers plan to initially set the price of honey very low and progressively increase this price to the point where it is sufficiently high to incentivize nodes to offer their service to the network while being low enough not to scare away potential users of Swarm. 
 * We expect the price of the counterpart currency to fluctuate in value against the costs of operating a Swarm node. As the real benefits (i.e. the worth of your compensation) of operating a Swarm node shouldn't fluctuate, the honey price should fluctuate inversely against the fluctuations of currency. If this happens, the real benefit stays equal, regardless of price fluctuations of the currency in which compensation is paid. 
```

Comment:
Again, I am not sure I agree to this. I vote to remove everything related to external currency fluctuations, I don't think we should do that. Please respond with arguments if it's indeed needed and why.

Reply:
Argument is given [above](https://github.com/Eknir/SWIPs/pull/9/files#r312818352). I propose to continue the discussion  there.

Reply:
Reading @holisticode comment I wonder if we actually need oracle for prices. If we can't use just normal services like conmarketcap, cryptocompare api etc... Because you never need to read this value in any smart contract hence no need to have oracle.

Reply:
I strongly argue against any reliance on Web 2.0 APIs and services.